### PR TITLE
ztp: support HTTP transport for Cloud Event Proxy

### PR DESCRIPTION
**Pull request by jzding**
Signed-off-by: Jack Ding <jackding@gmail.com>

### CODE REVIEW

File: ztp/policygenerator-kustomize-plugin/testPolicyGenTemplate/site-du-sno-1-ranGen.yaml [hunk 1]
```diff
@@ -2,12 +2,13 @@
 apiVersion: ran.openshift.io/v1
 kind: PolicyGenTemplate
 metadata:
-  name: "site-du-sno-1"
+  name: "site-du-sno-2"
```
[the remark this answers is not in the document]
Comment (nishant-parekh) [reply ztp/policygenerator-kustomize-plugin/testPolicyGenTemplate/site-du-sno-1-ranGen.yaml:5]: also update the commit message to include the `ztp:` prefix for the ci job failure

[the remark this answers is not in the document]
Comment (jzding) [reply ztp/policygenerator-kustomize-plugin/testPolicyGenTemplate/site-du-sno-1-ranGen.yaml]: Fixed file names to be consistent with the contents. I use HTTP transport for site-du-sno-1 and AMQP transport for site-du-sno-2, so we can just delete site-du-sno-2 in the future when AMQP transport is no longer supported.

**Review by nishant-parekh — APPROVED** [review]

**Pull request merged**